lab_1_keywords_tfidf/main.py

```diff
@@ -156,4 +230,17 @@ def extract_significant_words(chi_values: dict[str, float], alpha: float) -> Opt
 
     In case of corrupt input arguments, None is returned
     """
-    pass
+    if isinstance(chi_values, dict) and all(isinstance(token, str) for token in chi_values.keys()) and all(
+            isinstance(value, float) for value in chi_values.values()) and isinstance(alpha, float):
+        CRITERION = {0.05: 3.842, 0.01: 6.635, 0.001: 10.828}
```

Comment:
Why is this CAPS?

Reply:
that's how it was in the readme too! but I will replace it too